resolwe/flow/executors/docker.py

```diff
@@ -18,14 +18,18 @@ def __init__(self, *args, **kwargs):
         self.command = settings.FLOW_EXECUTOR.get('COMMAND', 'docker')
 
     def start(self):
-        container_image = settings.FLOW_EXECUTOR['CONTAINER_IMAGE']
+        # arguments passed to the Docker command
+        command_args = {
+            'command': self.command,
+            'container_image': settings.FLOW_EXECUTOR['CONTAINER_IMAGE'],
+        }
 
         if self.data_id != 'no_data_id':
-            container_name = 'resolwe_{}'.format(self.data_id)
+            data_id = self.data_id
         else:
             # set random container name for tests
-            rand_int = random.randint(1000, 9999)
-            container_name = 'resolwe_test_{}'.format(rand_int)
+            data_id = 'test_{}'.format(random.randint(1000, 9999))
```

Comment:
Yea, we could improve this. But since this is not a regression from what we have, I suggest to fix this later.